test/augmentation/test_augmentation_3d.py

```diff
@@ -18,980 +18,1098 @@
     RandomRotation3D,
     RandomCrop3D,
     CenterCrop3D,
-    RandomEqualize3D
+    RandomEqualize3D,
+    Normalize3D,
+    Denormalize3D,
 )
 
 
-class TestRandomHorizontalFlip3D:
+# class TestRandomHorizontalFlip3D:
```

Comment:
why all this code is commented ? we shouldn't include commented code

Reply:
Sorry, forgot to uncomment it out.